docs/src/explanations/connectable.md

```diff
@@ -0,0 +1,612 @@
+
+```scala mdoc:silent
+import chisel3.experimental.BundleLiterals._
+class Example4 extends RawModule {
+  val w = Wire(new MixedAlignmentBundle)
+  dontTouch(w) // So we see it in the output verilog
+  w :#= (new MixedAlignmentBundle).Lit(_.alignedChild -> true.B, _.flippedChild -> true.B)
+}
+```
+
+This generates the following Verilog, where all members are driven from the literal to `w`, regardless of alignment:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example4())
+```
+
+> Note: Astute observers will realize that semantically `c :#= p` is exactly equivalent to `c :<= p` followed by `p :>= c` (note `p` and `c` switched places in the second connection).
+
+Another use case for `:#=` is for connecting a mixed-directional bundle to a fully-aligned monitor.
+
+```scala mdoc:silent
+import chisel3.experimental.BundleLiterals._
+class Example4b extends RawModule {
+  val monitor = IO(Output(new MixedAlignmentBundle))
+  val w = Wire(new MixedAlignmentBundle)
+  dontTouch(w) // So we see it in the output verilog
+  w :#= DontCare
+  monitor :#= w
+}
+```
+
+This generates the following Verilog, where all members are driven from the literal to `w`, regardless of alignment:
+
+```scala mdoc:verilog
+import chisel3.stage.ChiselStage
+
+ChiselStage.emitVerilog(new Example4b())
+```
+## Waived Data
+
+It is not uncommon for a user to want to connect Chisel components which are not type equivalent. For example, a user may want to hook up anonymous `Record` components who may have an intersection of their fields being equivalent, but cannot because they are not structurally equivalent.
+
+`ConnectableData` is the mechanism to specialize connection operator behavior in these scenarios. For any additional member which is not present in the other component being connected to, they can be explicitly waived from the operator to be ignored, rather than trigger an error.
+
+In addition, there are other techniques that can be used to address similar use cases including `.viewAsSuperType`, a static cast to a supertype (e.g. `(x: T)`), or creating a custom dataview.
+For a discussion about when to use each technique, please continue [here](#techniques-for-connecting-structurally-inequivalent-chisel-types).
+
+This section demonstrates how `ConnectableData` specifically can be used in a multitude of scenarios.
+
+### Connecting Records
+
+A not uncommon usecase is to try to connect two Records; for matching members, they should be connected, but for unmatched members, they should be ignored. To accomplish this,  use the other operators to initialize all Record members, then use `:<>=` with `waiveAll` to connect only the matching members.
+
+> Note that `.viewAsSuperType`, static casts, nor a custom dataview helps this case because the types are still `Record`.
```

Comment:
```suggestion
> Note that none of `.viewAsSuperType`, static casts, nor a custom DataView helps this case because the Scala types are still `Record`.
```